src/management/token-provider.ts

```diff
@@ -114,7 +114,7 @@ export class TokenProvider {
 
       // TODO: Need to patch lru-memoizer to accept a max on its types.
       // max: 100,
-    })
+    }) as (options: TokenProviderOptions, callback: (err: any, tokenSet: TokenSet) => void) => void
```

Comment:
🤷 Couldnt get the types correct here, looks like some funkyness going on with the type returned by `memoizer`.